src/stub/src/arch/powerpc/64le/bxx.S

```diff
@@ -37,7 +37,7 @@ ppcbxx:  # (*f_unf)(xo->buf, out_len, h.b_cto8, h.b_ftid);
 #define ptr0 a4
 
         cmplwi cr0,ftid,0xd0;       bnelr- cr0  # if (0xd0!=ftid)   return;
-        rlwinm. len,len,32-2,2,31; beqlr- cr0  # if (0==(len>>=2)) return;
+        rldic. len,len,2,1; beqlr- cr0  # if (0==(len>>=2)) return;
```

Comment:
"rldicl. len,len,64-2,2" for "Rotate Left Double Immediate 64-2 bits (equivalent to Right 2 bits), Clear Left 2 bits, and record condition".  [Also "extrdi. len,len,64-2,0" for "EXTract and Right justify Double 64-2 bits beginning at bit 0 (the MSB), and record condition", but 'rldicl." is what appears in gdb.]   PowerPC-64 instructions for shift-and-mask are wacky enough that you MUST single-step them to verify.